app/helpers/articles_helper.rb

```diff
@@ -1,2 +1,23 @@
+require "zlib"
+
 module ArticlesHelper
+  WORDS_PER_MINUTE = 180
+
+  def article_reading_time(article)
+    words_count = article.content.split.size
+    minutes = (words_count.to_f / WORDS_PER_MINUTE).ceil
+    "#{minutes} min read"
+  end
+
+  ARTICLE_GRADIENTS = [
+    "from-[#98aaba] to-[#ffbd59]",
+    "from-[#fbc2eb] to-[#a6c1ee]",
+    "from-cyan-400 to-blue-500",
+    "from-[#f6d365] to-[#fda085]",
+    "from-[#f093fb] to-[#f5576c]"
+  ]
+
+  def article_gradient(article)
+    ARTICLE_GRADIENTS[Zlib.crc32(article.to_param) % ARTICLE_GRADIENTS.size]
```

Comment:
Gut reaction is that using zlib for this is overkill. Seems like there's probably a solution using built-in Ruby functionality.

Reply:
We could go for a simpler approach and just use the id of the article to select a gradient, the problem with that would be the gradients would have a repeating sequence as they're distributed one at a time.

Zlib is a part of the standard Ruby library, so there's no Gem's to install, the benefit of using it here is that gradients can be distributed in a somewhat random fashion to avoid weird looking screens where the gradients just repeat.

Happy to change to something else, like the following, if preferred though:
```ruby
ARTICLE_GRADIENTS[article.id % ARTICLE_GRADIENTS.size]
```

Reply:
Fair enough! Works for me.